.storybook/preview-head.html

```diff
@@ -0,0 +1,5 @@
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link
+    href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
```

Comment:
I might advice to not import all the weights in order to keep it as light as possible.
Maybe we can start with `400` (for default) & `500` (for primary and secondary) for now then see it we need something else later.

Reply:
As discussed, I explored and implemented the tools offered by Storybook. In the following list, you can check which ones I implemented and why. Also, notice that there were some tools that I didn't implement.

[Test runner](https://storybook.js.org/docs/react/writing-tests/test-runner) IMPLEMENTED. I implemented it because it lets you run all the tests that you wrote in the play function of every story. It will be interesting to add it to the building process in GitHub and in prod. Check the PR description to learn how to run it.

[Visual tests](https://storybook.js.org/docs/react/writing-tests/visual-testing) I didn't implement since it requires registering for a special token, and I don't have the credentials of our team to do such a thing. Also, I'm not sure how useful it will be for this particular component. TO BE DISCUSSED.

[Accessibility tests](https://storybook.js.org/docs/react/writing-tests/accessibility-testing) IMPLEMENTED. I implemented it since it was easy to implement, and it might be really helpful for the dev experience.

[Interaction tests](https://storybook.js.org/docs/react/writing-tests/interaction-testing) IMPLEMENTED. I implemented it since this is the core of the Storybook testing approach. Specifically for the button component, I focused on the styles of every variant.

[Coverage tests](https://storybook.js.org/docs/react/writing-tests/test-coverage) IMPLEMENTED. Coverage tests are useful because they ensure that the code is thoroughly tested, identify areas where tests are lacking, and can help optimise code. 

[Snapshot tests](https://storybook.js.org/docs/react/writing-tests/snapshot-testing) DIDN'T IMPLEMENT. It requires installing the JEST testing library.

[Import stories in other tests](https://storybook.js.org/docs/react/writing-tests/importing-stories-in-tests) DIDN'T IMPLEMENT. Since it's more related to test libraries, and my goal wasn't to write Cypress tests for now.